src/index.js

```diff
@@ -79,8 +84,12 @@ async function bundleMDX(
       })
 
       build.onLoad(
-        {filter: /__mdx_bundler_fake_dir__/},
-        async ({path: filePath}) => {
+        {filter: /.*/},
+        async ({path: filePath, pluginData}) => {
+          if(pluginData === undefined || !pluginData.inMemory){
```

Comment:
This is just a nit-pick (I really should figure out our git hooks so we can have prettier automatic). You don't have prettier installed in your editor?

Reply:
I don't unfortunatley, although I have run `npx kcd-scripts format` before comitting to tidy this up. Will run it now and tidy things up.